packages/block-library/src/cover/style.scss

```diff
@@ -17,6 +16,16 @@
 	// Keep the flex layout direction to the physical direction (LTR) in RTL languages.
 	/*rtl:raw: direction: ltr; */
 
+	// When an aspect ratio is set, the height is calculated based on the width, so don't
+	// manually set a min-height.
+	&:where(:not(.has-aspect-ratio)) {
+		min-height: 430px;
+	}
+
+	&:where(.has-aspect-ratio) {
+		width: 100%;
+	}
```

Comment:
We'll likely need to put this rule somewhere else as I think we'll always want this applied when `.has-aspect-ratio` is active.